labels.md

```diff
@@ -0,0 +1,220 @@
+proposal                        #eb6420     type/proposal                       #1d76db
+question                        #fbca04     type/question                       #1d76db
+release-note                    #1d76db     * <type/CHANGELOG>                    
+require-LGT3                    #d93f0b     type/require 3 reviews              #1d76db
+status/DNM                      #b60205     status/DNM                          #e6e6e6
+status/LGT1                     #d4c5f9     * <status/Waiting on review>            
+status/LGT2                     #5319e7     * <status/Waiting on review>            
+status/LGT3                     #330099     * <status/Waiting on review>            
+status/PTAL                     #2cbe4e     status/waiting on review            #e6e6e6
+status/ReqChange                #b60205     status/waiting on author            #e6e6e6
+status/TODO                     #207de5     *                                   
+status/WIP                      #fbca04     status/WIP                          #e6e6e6
+status/all tests passed         #2cbe4e     * <status/bot merge>                    
+status/can merge                #2cbe4e     status/bot merge                    #e6e6e6
+status/future                   #fbca04     ?                                   
+type/1.0 cherry-pick            #c2e0c6     *                                   
+type/2.0 cherry-pick            #99d0ef     *                                   
+type/2.1 cherry-pick            #9bfff5     *
+type/3.0                        #ededed     *                                   
+type/3.0 cherry-pick            #297b99     *
+type/TEP                        #bfd4f2     type/TEP                            #1d76db
+type/bug                        #fc2929     type/bug                            #1d76db
+type/bug-fix                    #fc2929     type/bug fix                        #1d76db
+type/compatibility              #e99695     type/compatibility                  #1d76db
+type/duplicate                  #cccccc     status/closed dup                   #e6e6e6
+type/enhancement                #84b6eb     type/enhancement                    #1d76db
+type/invalid                    #e6e6e6     status/closed invalid               #e6e6e6
+type/investigation              #f46100     type/investigation                  #1d76db
+type/new-feature                #e2e876     * <type/enhancement>
+type/performance                #006b75     type/performance                    #1d76db
+type/regression                 #c41b2f     type/regression                     #1d76db
+type/suggestion                 #11b2c4     * <type/proposal>                     
+type/wontfix                    #95d3db     * <just close the issue>            
+v3.0                            #cb11f9     *                                   
+```
+
+## TiKV
+
+```
+C: Build                        #d1fad7     component/build                     #d1fad7
```

Comment:
Looks like these two conflicts to each other: "We use a `C: label` scheme for all repos" "Current labels and colours are on the left, proposed are on the right"